code-samples/eventing/bookstore-sample-app/slack-sink/slack-sink-camel-pipe.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: camel.apache.org/v1
+kind: Pipe
+metadata:
+  name: slack-sink-pipe
+spec:
+  source:
+    ref:
+      kind: Broker
+      apiVersion: eventing.knative.dev/v1
+      name: book-review-broker
+    properties:
+      type: new-review-comment
+  sink:
+    ref:
+      kind: Kamelet
+      apiVersion: camel.apache.org/v1
+      name: slack-sink
+    properties:
+      channel: “#bookstore-owner”
+      webhookUrl: <Your Slack Web Hook to the channel above>
```

Comment:
@Leo6Leo what's the purpose of this file? Don't we have this same yaml in the README?

Reply:
Yes, it is the same content as the one in the README. I just have all the code in files as well, so readers can directly download all the code and run.

Reply:
Great catch!! Thanks @Cali0707 ! I didn't realize that they are inconsistent.